docs/spec/circuit-breaker/01_concepts.md

```diff
@@ -0,0 +1,17 @@
+# Concepts
+
+The intention of the circuit breaker is to have a contingency plan for a
+running network which maintains network liveness. This can be achieved through
+selectively "pausing" functionality of specific modules on a running network.
+The circuit breaker is intended to be enabled through either:
+
+ - governance, 
+ - the bonded validator group (for emergencies), 
```

Comment:
@cwgoes (responding in a different conversation) 

> how is this different than the bonded validator set just stopping the consensus process?

This will have to be detailed in the actual spec, however, I imagine in makes sense for there to be consensus _among_ the validator set through either config or some alternative special validator only consensus process which shuts off a message type for all validators (so now if you're a validator who had not been aware of the circuit breaker, but you received the information that the circuit breaker had been switched, you would nolonger accept those messages in blocks or by CheckTx)